app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/data/GetInsuranceContractsUseCase.kt

```diff
@@ -36,19 +36,19 @@ internal class GetInsuranceContractsUseCaseImpl(
       featureManager.isFeatureEnabled(Feature.EDIT_COINSURED),
       featureManager.isFeatureEnabled(Feature.MOVING_FLOW),
       featureManager.isFeatureEnabled(Feature.TIER),
-    ) { insuranceQueryResponse, isEditCoInsuredEnabled, isMovingFlowEnabled, isTierEnabled ->
+    ) { insuranceQueryResponse, isEditCoInsuredEnabled, isMovingFlowFlagEnabled, isTierEnabled ->
       either {
         val insuranceQueryData = insuranceQueryResponse.bind()
         val contractHolderDisplayName = insuranceQueryData.getContractHolderDisplayName()
         val contractHolderSSN = insuranceQueryData.currentMember.ssn?.let { formatSsn(it) }
-
+        val isMovingEnabledForMember = insuranceQueryData.currentMember.memberActions?.isMovingEnabled ?: false
```

Comment:
We can probably do all of this in one go here, so we can use just `isMovingEnabledForMember` in the two places below, without risking accidentally changing one of them but forgetting to change the other, for example.
```suggestion
        val isMovingEnabledForMember = insuranceQueryData.currentMember.memberActions?.isMovingEnabled == true && isMovingFlowFlagEnabled
```